spec/support/fake_authentic_jobs.rb

```diff
@@ -0,0 +1,20 @@
+require 'sinatra/base'
+
+class FakeAuthenticJobs < Sinatra::Base
+  get "/api/" do
+    if params[:keywords] == 'ruby'
+      json_response 200, 'authentic_jobs_ruby_response.json'
+    elsif params[:keywords] == 'rubular'
+      json_response 200, 'authentic_jobs_no_results_response.json'
+    elsif params[:api_key] == '123'
+      json_response 200, 'authentic_jobs_bad_api_key_response.json'
+    end
```

Comment:
nbd but I'd opt for a case statement here.  I'm an if elsif else tops kinda guy.